Sample Data/Custom/AliCloud_CL.json

```diff
@@ -0,0 +1,182 @@
+   "Type": "AliCloud_CL",
+   "_ResourceId": ""
+ },
+ {
+   "TenantId": "9143fd29-fe92-43be-93e9-3f0a4bcaeef4",
+   "SourceSystem": "RestAPI",
+   "MG": "",
+   "ManagementGroupName": "",
+   "TimeGenerated [UTC]": "5/4/2022, 10:40:39.087 PM",
+   "Computer": "",
+   "RawData": "",
+   "contents___tag_____pack_id___s": "",
+   "contents_event_s": "{\"acsRegion\":\"cn-hangzhou\",\"additionalEventData\":{\"CallerBid\":\"26842\"},\"eventCategory\":\"Management\",\"eventId\":\"627116E9C2A6D274715D3093\",\"eventName\":\"ListProject\",\"eventRW\":\"Read\",\"eventSource\":\"cn-hangzhou.log.aliyuncs.com\",\"eventTime\":\"2022-05-03T11:50:01Z\",\"eventType\":\"ApiCall\",\"eventVersion\":\"1\",\"requestId\":\"627116E9C2A6D274715D3093\",\"requestParameterJson\":\"{\\\"offset\\\":\\\"0\\\",\\\"size\\\":\\\"500\\\"}\",\"requestParameters\":{\"offset\":\"0\",\"size\":\"500\"},\"serviceName\":\"SLS\",\"sourceIpAddress\":\"77.77.77.77\",\"userAgent\":\"log-python-sdk-v-0.7.7, sys.version_info(major=3, minor=8, micro=12, releaselevel='final', serial=0), linux\",\"userIdentity\":{\"accessKeyId\":\"LTAI5tChhsEpyPJdPqXxDyNN\",\"accountId\":\"5558349267652842\",\"principalId\":\"5558349267652842\",\"sessionContext\":{\"attributes\":{\"mfaAuthenticated\":\"false\"}},\"type\":\"root-account\",\"userName\":\"root\"}}",
+   "timestamp_d": 1651578601,
+   "source_s": "actiontrail_internal",
+   "contents_key_1_s": "",
+   "contents_avg_s": "",
+   "contents___topic___s": "actiontrail_audit_event",
+   "Type": "AliCloud_CL",
+   "_ResourceId": ""
+ },
+ {
+   "TenantId": "9143fd29-fe92-43be-93e9-3f0a4bcaeef4",
+   "SourceSystem": "RestAPI",
+   "MG": "",
+   "ManagementGroupName": "",
+   "TimeGenerated [UTC]": "5/4/2022, 10:40:39.087 PM",
+   "Computer": "",
+   "RawData": "",
+   "contents___tag_____pack_id___s": "",
+   "contents_event_s": "{\"acsRegion\":\"cn-hangzhou\",\"additionalEventData\":{\"CallerBid\":\"26842\"},\"eventCategory\":\"Management\",\"eventId\":\"6271194166F8292189C9A43C\",\"eventName\":\"ListProject\",\"eventRW\":\"Read\",\"eventSource\":\"cn-hangzhou.log.aliyuncs.com\",\"eventTime\":\"2022-05-03T12:00:01Z\",\"eventType\":\"ApiCall\",\"eventVersion\":\"1\",\"requestId\":\"6271194166F8292189C9A43C\",\"requestParameterJson\":\"{\\\"offset\\\":\\\"0\\\",\\\"size\\\":\\\"500\\\"}\",\"requestParameters\":{\"offset\":\"0\",\"size\":\"500\"},\"serviceName\":\"SLS\",\"sourceIpAddress\":\"77.77.77.77\",\"userAgent\":\"log-python-sdk-v-0.7.7, sys.version_info(major=3, minor=8, micro=12, releaselevel='final', serial=0), linux\",\"userIdentity\":{\"accessKeyId\":\"LTAI5tChhsEpyPJdPqXxDyNN\",\"accountId\":\"5558349267652842\",\"principalId\":\"5558349267652842\",\"sessionContext\":{\"attributes\":{\"mfaAuthenticated\":\"false\"}},\"type\":\"root-account\",\"userName\":\"root\"}}",
+   "timestamp_d": 1651579201,
+   "source_s": "actiontrail_internal",
+   "contents_key_1_s": "",
+   "contents_avg_s": "",
+   "contents___topic___s": "actiontrail_audit_event",
```

Comment:
Sample data columns should not be specified with _d and _s.  With this sample data ingestion, parser query throwing the error as it was unable to find the column names provided in the parser text.  for instance, in the sample data "timestamp_d" is defined and in the loganalytics column name was created as "timestamp_d_d". so after running the parser query getting the below error. Please validate the column names and provide updated sample data. 
![image](https://user-images.githubusercontent.com/105280229/171399691-7e4bb0fe-4a0e-4ccb-ac1c-5f081c6248bc.png)
![image](https://user-images.githubusercontent.com/105280229/171399858-04a96fe4-07cd-4031-99e7-7594765c0ff1.png)

Reply:
@VenkataPrathibha, returned initial sample data

Reply:
@KateShapovalova, It was throwing again the error. Is it possible just to update the column names with the la last json provided. 
![image](https://user-images.githubusercontent.com/105280229/172108333-aad7b0f7-0623-453e-9e0d-73109b670ae3.png)
Did you validate from your end. Kindly provide the steps how did you validate.

Reply:
@KateShapovalova, It was throwing again the error. Is it possible just to update the column names with the la last json provided. 
![image](https://user-images.githubusercontent.com/105280229/172108333-aad7b0f7-0623-453e-9e0d-73109b670ae3.png)
Did you validate from your end. Kindly provide the steps how did you validate.

Reply:
@VenkataPrathibha, added a sample with this column. For validation: I connect to AliCloud by means of AliCloud API to get the sample data. Then push it to Azure Sentinel and run the parse script.